lib/docker-entrypoint-webpack.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# https://stackoverflow.com/a/38732187/1935918
+
+cd /app
+bundle install
+yarn install
```

Comment:
This is so we don't need to rebuild the whole image on a gem update or yarn.lock update.